src/main/java/org/gridsuite/filter/server/dto/criteriafilter/AbstractLineFilter.java

```diff
@@ -4,39 +4,32 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package org.gridsuite.filter.server.dto;
-
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.SortedSet;
-
-import org.gridsuite.filter.server.utils.EquipmentType;
-import org.springframework.util.CollectionUtils;
+package org.gridsuite.filter.server.dto.criteriafilter;
 
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 import lombok.ToString;
 import lombok.experimental.SuperBuilder;
+import org.springframework.util.CollectionUtils;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedSet;
 
 /**
- * @author Jacques Borsenberger <jacques.borsenberger at rte-france.com>
+ * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
```

Comment:
If considering this abstract class as new => change Copyright (c) 2021 to Copyright (c) 2023